### ISSUE TITLE
Problem adding two different products, but with the same color and size

### ISSUE THREAD

**Issue by alinadivante** [DivanteLtd/vue-storefront#4132]
## Current behavior
If the user adds two different SKUs to the cart, but with the same color and size, only Qty of the first product will be increased, and only one of the products will be in the cart.
![image](https://user-images.githubusercontent.com/36628922/75519027-984f7500-5a02-11ea-8f2e-cc9ceaf4f9b2.png)

## Expected behavior
Two products should be added to the cart.

## Steps to reproduce the issue
1. Add for example this product http://localhost:3000/men/bottoms-men/shorts-men/shorts-19/cobalt-cooltech-and-trade-fitness-short-898.html?childSku=MSH01
choosing color: **blue** and size: **33**
2. Add another product, e.g http://localhost:3000/men/bottoms-men/shorts-men/shorts-19/troy-yoga-short-994.html?childSku=MSH09
and also choose **blue** and size: **33**

## Can you handle fixing this bug by yourself?

- [ ] YES
- [ ] NO

## Which [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html) state this refers to? Info for developer.
Pick one option.

- [ ] This is a bug report for test version on https://test.storefrontcloud.io - In this case Developer should create branch from `develop` branch and create Pull Request `2. Feature / Improvement` back to `develop`.
- [ ] This is a bug report for current Release Candidate version on https://next.storefrontcloud.io - In this case Developer should create branch from `release` branch and create Pull Request `3. Stabilisation fix` back to `release`.
- [ ] This is a bug report for current Stable version on https://demo.storefrontcloud.io and should be placed in next stable version hotfix - In this case Developer should create branch from `hotfix` or `master` branch and create Pull Request `4. Hotfix` back to `hotfix`.

## Environment details

- Browser: 
- OS: 
- Node: 
- Code Version:

**Comment by pkarw** [comment]
This is urgent and we probably need to release 1.11.3 just with this one @andrzejewsky @patzick @gibkigonzo

**Comment by gibkigonzo** [comment]
We can add this to 1.11.2 https://github.com/DivanteLtd/vue-storefront/blob/hotfix/v1.11.2/core/modules/cart/helpers/productsEquals.ts#L81 it requires change to:
```
// if is same configurable option and sku
return check(['checksum']) && check(['sku'])
```
I will fix this as soon as possible

**Comment by andrzejewsky** [comment]
Yeah, really thanks @gibkigonzo - It could be in 1.11.2, better to wait for one (important though) fix, actually 1.11.2 is still in processing by QA

**Comment by pkarw** [comment]
@andrzejewsky got it - it's a draft (the release). Sure, lets be sure that cart is fixed in the 1.11.2 100%
